lib/tasks/sidekiq.rake

```diff
@@ -18,7 +18,7 @@ namespace :sidekiq do
 
     args = []
     args.push(['--index', ENV.fetch('INDEX', '0')])
-    args.push(%w[backend_sync billing critical default deletion events low mailers priority web_hooks zync].flat_map { |queue| ['--queue', queue] })
+    args.push(%w[backend_sync billing critical default deletion events low mailers non_critical priority web_hooks zync].flat_map { |queue| ['--queue', queue] })
```

Comment:
I'd rather explicitly name it `sphinx_indexation` or `indexation`, since it will already be added to `sidekiq_low_prio_queues`.